src/System.Net.Ping/src/System/Net/NetworkInformation/Ping.Unix.cs

```diff
@@ -175,9 +175,9 @@ public partial class Ping
             else
             {
                 cts.Cancel();
-                if (p.ExitCode != 0)
+                if (p.ExitCode == 1 || p.ExitCode == 2)
```

Comment:
SInce you say these are known, can you drop a hint as to where you found them and what they mean?